docs/Community/governance/OWNERS

```diff
@@ -0,0 +1,7 @@
+chair:
+
+reviewers:
```

Comment:
As a general comment, can we order all of the OWNERS in responsibility level as described in the sig-charter-template.md? It also provides the opportunity drop empty categories from files that don't need them yet (e.g. emeritus, member)

Reply:
done